README.md

```diff
@@ -40,15 +40,15 @@ We've added additional Ruby gems and other software that aren't automatically av
 - Fly.io's `flyctl`
 - Google Chrome (headless browser)
 - Graphviz
-- Heroku 
 - Node JS 18
 - NPM 8.19.3
-- Parity
```

Comment:
I should have removed these two things from the readme on https://github.com/appdev-projects/rails-7-template/commit/03a522a17ae92bc4af94e75da9b9873ba7f410a9. Minor update, so doing it here.